scripts/run.sh

```diff
@@ -10,6 +13,9 @@ build_front_end() {
 run_back_end_tests() {
   cd /app
   tox $@
+  echo 'repo_token:' $repo_token >> .coveralls.yml
```

Comment:
No, just the repo_token is all that is needed in the `.coveralls.yml` file. When the `coveralls` command is run, it looks for the file and gets the token.